torch/csrc/autograd/functions/convolution.cpp

```diff
@@ -257,63 +255,63 @@ auto ConvBackward::apply(const variable_list& grad_outputs) -> variable_list {
   auto weight_var = weight_.unpack();
   auto bias_var = bias_.unpack();
 
-  std::unique_ptr<thpp::Tensor> input {input_var->data->clone_shallow()};
-  std::unique_ptr<thpp::Tensor> weight {weight_var->data->clone_shallow()};
-  std::unique_ptr<thpp::Tensor> bias {bias_var ? bias_var->data->clone_shallow() : nullptr};
+  auto input = input_var->data;
+  auto weight = weight_var->data;
+  auto bias = bias_var ? bias_var->data : at::Tensor();
```

Comment:
I don't really understand. Isn't view supposed to work out of place? If so, having const references should be ok (as long as ATen is const correct). Anyway, I'd rather merge this fast, so we can leave it as is